### PR TITLE
Improve syslog rate limit feature for warm/fast reboot

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: dockers/docker-database/supervisord.conf.j2 [hunk 1]
```diff
@@ -39,7 +39,7 @@ dependent_startup=true
 command=/bin/bash -c "{ [[ -s /var/lib/{{ redis_inst }}/dump.rdb ]] || rm -f /var/lib/{{ redis_inst }}/dump.rdb; } && mkdir -p /var/lib/{{ redis_inst }} && exec /usr/bin/redis-server /etc/redis/redis.conf --bind {{ LOOPBACK_IP }} {{ redis_items['hostname'] }} --port {{ redis_items['port'] }} --unixsocket {{ redis_items['unix_socket_path'] }} --pidfile /var/run/redis/{{ redis_inst }}.pid --dir /var/lib/{{ redis_inst }}"
 priority=2
 user=redis
-autostart=false
+autostart=true
```
Comment (stepanblyschak) [inline dockers/docker-database/supervisord.conf.j2:42]: Delete dependent_startup=true and dependent_startup_wait_for=rsyslogd:running as well

File: files/image_config/rsyslog/rsyslog-container.conf.j2 [hunk 2]
```diff
@@ -28,12 +28,12 @@ $ModLoad imuxsock # provides support for local system logging
 
 {% if rate_limit_interval is defined %}
 $SystemLogRateLimitInterval {{ rate_limit_interval }}
-{% else %}
+{% else %}
```
Comment (stepanblyschak) [inline files/image_config/rsyslog/rsyslog-container.conf.j2:31]: Remove extra space

File: files/image_config/rsyslog/rsyslog-container.conf.j2 [hunk 3]
```diff
@@ -28,12 +28,12 @@ $ModLoad imuxsock # provides support for local system logging
...
 {% if rate_limit_interval is defined %}
 $SystemLogRateLimitInterval {{ rate_limit_interval }}
-{% else %}
+{% else %}	
 $SystemLogRateLimitInterval 300
 {% endif %}
 {% if rate_limit_burst is defined %}
 $SystemLogRateLimitBurst {{ rate_limit_burst }}
-{% else %}
+{% else %}
```
Comment (stepanblyschak) [inline files/image_config/rsyslog/rsyslog-container.conf.j2:36]: Remove extra space

File: src/sonic-containercfgd/containercfgd/containercfgd.py [hunk 4]
```diff
@@ -1,11 +1,14 @@
+from swsscommon.swsscommon import RestartWaiter
+RestartWaiter.waitAdvancedBootDone()
```
Comment (stepanblyschak) [inline src/sonic-containercfgd/containercfgd/containercfgd.py:2]: Does it make any measured difference?  Can we just sleep in supervisord command line ?

Comment (Junchao-Mellanox) [reply src/sonic-containercfgd/containercfgd/containercfgd.py:2]: In theory, it should

**Comment by Junchao-Mellanox** [comment]
Opened new PR, closing this one

**Pull request closed without merging**